lib/cc/analyzer/engine_output_filter.rb

```diff
@@ -22,7 +22,7 @@ def check_disabled?(issue)
       end
 
       def ignore_fingerprint?(issue)
-        @config.fetch("exclude_fingerprints", []).include?(issue.fingerprint)
+        @config.fetch("exclude_fingerprints", []).include?(issue.fingerprint(expect_valid_locations: false))
```

Comment:
that sounds reasonable. The reason it felt a little weird was that the exception is pretty far away from this call site. For this call site to know that that exception might be raised requires it to know that `Issue#fingerprint` eventually, sometimes uses `SourceExtractor`. I guess that's OK, though.. and does make a smaller diff... I think I'm convinced, will update :)